modules/network_synchronizer/scene_synchronizer.h

```diff
@@ -324,15 +324,15 @@ class Synchronizer {
 
 public:
 	Synchronizer(SceneSynchronizer *p_node);
-	virtual ~Synchronizer();
```

Comment:
I prefer leave the implementation, even if void inside the CPP file, so here we can just have the definition.